py/desisim/scripts/quickquasars.py

```diff
@@ -231,17 +231,19 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
 
     elif(args.dla=='random'):
         log.info('Adding DLAs randomly')
+        saved_rnd_state = np.random.get_state()
```

Comment:
Its much easier to manage different parts of a program consuming random numbers independently if they each have their own private `np.random.RandomState`.  A corollary is that you should avoid using the global RandomState with direct calls to `np.random....`